app/serializers/move_serializer.rb

```diff
@@ -29,6 +29,7 @@ class MoveSerializer < ActiveModel::Serializer
 
   SUPPORTED_RELATIONSHIPS = %w[
     profile
+    person
```

Comment:
instead of having to explicitly support this everywhere going forward (as now we will be rolling this out to all relationships) can we fix the validation itself to check for anything before the `.` and deem it valid?

Reply:
We definitely could. Maybe we should solve this in the generalisation work we can refine in today's refinement.

Reply:
I guess it's recursive, too. So all of the levels - not just the root - get included. Simple test will prove/disprove this